evidence/create_package.go

```diff
@@ -0,0 +1,129 @@
+package evidence
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/jfrog/jfrog-cli-artifactory/evidence/model"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	coreConfig "github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-client-go/artifactory"
+	"github.com/jfrog/jfrog-client-go/artifactory/services"
+	"github.com/jfrog/jfrog-client-go/metadata"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+const leadArtifactQueryTemplate = `{
```

Comment:
Why putting a const if you dont use it as const?
Also, it is used only in one place just copy it to line 124

Reply:
i tried to change it, but i think also is not bad to have it private const here since its too long and a bit confusing to have it directly inside the small method.